python/ray/autoscaler/_private/prom_metrics.py

```diff
@@ -202,6 +202,22 @@ def __init__(self, registry: Optional[CollectorRegistry] = None):
                 namespace="autoscaler",
                 registry=self.registry,
             )
+            self.cluster_resources: Counter = Gauge(
```

Comment:
What does this mean this is a Counter type, but it is actually a Gauge?

Reply:
self.cluster_resources: **Counter** = **Gauge**(

Reply:
you would think mypy would've complained about that lol